Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.5.1
+
+WORKDIR /app
+COPY . /app
+
+RUN pip install --upgrade --no-cache-dir -r app/requirements.txt \
+    && python setup.py install
```

Comment:
I'm unable to get `docker build .` to run successfully with this Dockerfile.

It looks like pylibmc compilation fails because the libmemcached headers are missing from the VM: https://6a68.pastebin.mozilla.org/8855787

Do you want to debug this, or just rip the Dockerfile out for now?